docker/php/apache/run.sh

```diff
@@ -15,8 +15,16 @@ if [ -z "${PHP_AGENT_VERSION}" ] ; then
   make clean install
   cp /src/ext-elastic-apm.ini /usr/local/etc/php/conf.d/ext-elastic-apm.ini
 else
-  ## Install the given release
-  wget -q "https://github.com/elastic/apm-agent-php/releases/download/v${PHP_AGENT_VERSION}/apm-agent-php_${PHP_AGENT_VERSION}_all.deb" \
+  if [ "${PHP_AGENT_VERSION}" == "latest" ]  ; then
+    wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+    chmod 755 jq
+    TAG_NAME=$(curl -s https://api.github.com/repos/elastic/apm-agent-php/releases/latest | jq -r .tag_name)
```

Comment:
```suggestion
    TAG_NAME=$(curl -s https://api.github.com/repos/elastic/apm-agent-php/releases/latest | ./jq -r .tag_name)
```
I'm surprised this doesn't need a `./` here if it's downloaded into the current dir. Are you expecting to already be in a directory that's in the $PATH?

Reply:
`[2020-11-16T12:50:23.753Z] ./run.sh: line 21: jq: command not found` but the shell doesn't stop afterwards :/